codecov.yml

```diff
@@ -0,0 +1,7 @@
+comment:
+  layout: "diff"
+  behavior: default
+  branches:
+    - master
+    - develop
```

Comment:
I don't think it will disable comments for PR, this controls which branches should be considered while commenting, i.e. if the head of the pull request is master or develop, then only comments will work. https://docs.codecov.io/docs/pull-request-comments#branches

I think we can remove this, as PR are mostly headed to the develop branch.

Reply:
Okey, let's keep it as it is and we will see how it works